pandas/tests/frame/test_indexing.py

```diff
@@ -1688,6 +1688,20 @@ def test_getitem_list_duplicates(self):
         expected = df.iloc[:, 2:]
         assert_frame_equal(result, expected)
 
+    def test_getitem_setitem_datetimeindex_tz(self):
```

Comment:
I think there are other tests like this, can you co-locate (IIRC in same file), *maybe* though in pandas/tests/indexing/....